src/telemetry/telemetryService.ts

```diff
@@ -49,18 +49,8 @@ type DocumentEditedTelemetryEventProperties = {
   source: DocumentSource;
 };
 
-type AggregationExportedTelemetryEventProperties = {
+type PlaygroundExportedToLanguageTelemetryEventProperties = {
   language: string;
-  num_stages: number | null;
-  with_import_statements: boolean;
-  with_builders: boolean;
-  with_driver_syntax: boolean;
-};
-
-type QueryExportedTelemetryEventProperties = {
-  language: string;
-  with_import_statements: boolean;
-  with_builders: boolean;
   with_driver_syntax: boolean;
```

Comment:
Do you mean `num_stages`? We can export now not only arrays and objects but any code from a playground, so a number of stages value is not relevant anymore. I think I can add a new parameter to track simply the length of the exported string. It will give us a rough understanding of how much code users export.

Reply:
I was just thinking the length of the string yup.